ODLA/platforms/dnnl/dnnl_utils_avx512.h

```diff
@@ -144,14 +118,14 @@ inline void floorbf_func(int len, float* src, float* dst) {
     auto out1 = _mm512_floor_ps(a1);
 
     auto C_bf16 = _mm512_cvtne2ps_pbh(out1, out0);
-    _mm512_mask_storeu_ps(dst + i, mask16, _mm512_castsi512_ps(C_bf16));
+    _mm512_mask_storeu_ps(dst + i / 2, mask16, _mm512_castsi512_ps(C_bf16));
```

Comment:
should this still be "dst + i"?

Reply:
dst is float* and vec_size is 32, so we should store i/2 vec_size (512bit) in single loop :)